pinot-server/src/test/java/com/linkedin/pinot/server/api/resources/TablesResourceTest.java

```diff
@@ -190,4 +185,13 @@ public void testSegmentCrcMetadata() throws Exception {
       assertEquals(crcMap.get(segmentName), crc);
     }
   }
+
+  private Map<String, String> fetchCrcInformation(String urlFormat) throws Exception {
+    // Trigger crc api to fetch crc information
+    String response = testHelper.target.path(String.format(urlFormat, ResourceTestHelper.DEFAULT_TABLE_NAME))
+        .request().get(String.class);
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Make ObjectMapper a private static final member variable. Also, change the import to use codehaus one.